src/client.rs

```diff
@@ -731,6 +819,99 @@ mod test {
             .unwrap();
         let expected = vec![ImportDescriptorResult { success: true }];
         assert_eq!(expected, got);
+
+        let psbt_address = client.get_new_address().await.unwrap();
+        let mut psbt_outputs = std::collections::HashMap::new();
+        psbt_outputs.insert(psbt_address.to_string(), 1.0);
+
+        let funded_psbt = client
+            .wallet_create_funded_psbt(&[], &psbt_outputs, None, None, None)
+            .await
+            .unwrap();
+        assert!(!funded_psbt.psbt.inputs.is_empty());
+        assert!(funded_psbt.fee.to_sat() > 0);
+
+        let processed_psbt = client
+            .wallet_process_psbt(&funded_psbt.psbt.to_string(), None, None, None)
+            .await
+            .unwrap();
+        assert!(!processed_psbt.psbt.as_ref().unwrap().inputs.is_empty());
+        assert!(processed_psbt.complete);
+
+        let finalized_psbt = client
+            .finalize_psbt(&processed_psbt.psbt.as_ref().unwrap().to_string(), Some(true))
+            .await
+            .unwrap();
+        assert!(finalized_psbt.complete);
+        assert!(finalized_psbt.hex.is_some());
+
+        let info_address = client.get_new_address().await.unwrap();
+        let address_info = client.get_address_info(&info_address).await.unwrap();
+        assert_eq!(address_info.address, info_address.as_unchecked().clone());
+        assert!(address_info.is_mine.unwrap_or(false));
+        assert!(address_info.solvable.unwrap_or(false));
+
+        let unspent_address = client.get_new_address().await.unwrap();
+        let unspent_txid = client
+            .call::<String>(
+                "sendtoaddress",
+                &[
+                    to_value(unspent_address.to_string()).unwrap(),
+                    to_value(1.0).unwrap(),
+                ],
+            )
+            .await
+            .unwrap();
+        mine_blocks(&bitcoind, 1, None).unwrap();
+
+        let utxos = client
+            .list_unspent_with_options(Some(1), Some(9999999), None, Some(true), None)
+            .await
+            .unwrap();
+        assert!(!utxos.is_empty());
+
+        let utxos_filtered = client
+            .list_unspent_with_options(
+                Some(1),
+                Some(9999999),
```

Comment:
Please add `_` thousands separator

```suggestion
                Some(9_999_999),
```